apps/mobile/src/hooks/useOpenURL.js

```diff
@@ -25,18 +26,6 @@ export async function openURL (providedPathOrURL, reset, navigation = navigation
   ) {
     const linkingPath = linkingURL.pathname + linkingURL.search
 
-    if (DEBUG) {
```

Comment:
Can we just comment this out, for future debugging aid?

Reply:
The pathMatcher is now part of the routeParams, so is already logged in the matched state object which was the primary thing achieved by this extra debugging.